internal/proto/reader_test.go

```diff
@@ -11,6 +12,10 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+const (
+	bufioDefaultBufSize = 4096
```

Comment:
Bufio doesn't export this value, I had to copy it. I hope it won't change often.